src/Notify/index.js

```diff
@@ -47,8 +53,9 @@ export default class Notify extends Component {
         onClose={this.handleClick}
       >
         <div className={className}>
-          {notifications.map(({ type, message, id, isSticky, timeout, icon }) => (
```

Comment:
instead of removing `isSticky` and `timeout` I think we should pass them to the notification.

It doesn't make sense to specify on the container level that every single notification will be sticky, you're likely to only have a few notifications that are sticky and you need them to be able to specify that without having to setup a separate container for those kinds of notifications.
And notifications should also be able to override their timeout individually.